AdyenCard/Form/FormCardExpiryDateItem.swift

```diff
@@ -0,0 +1,63 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+
+/// A form item into which card expiry date is entered, formatted and validated.
+internal final class FormCardExpiryDateItem: FormTextItem, Hidable {
+    
+    /// :nodoc:
+    public var isHidden: Observable<Bool> = Observable(false)
+    
+    /// :nodoc:
+    internal var localizationParameters: LocalizationParameters?
+    
+    /// Flag determining this forms state. Validation changes based on this.
+    internal var isOptional: Bool = false {
+        didSet {
+            updateFormState()
+        }
+    }
+    
+    private lazy var expiryDateValidator = CardExpiryDateValidator()
```

Comment:
we probably don't need this line, we can do the assignment in line 36

Reply:
One of them is accessed later, so it is needed. The other one I kept for similarity. That one can be removed.

Reply:
alright. can it be a `let` instead of a `lazy var`? :) or maybe a `static let`?